boxwise_flask/models/qr.py

```diff
@@ -0,0 +1,20 @@
+from peewee import CharField, IntegerField
+
+from ..db import db
+
+
+class Qr(db.Model):
+    id = IntegerField()
```

Comment:
explicit id field. I'll do some research into why I had issues with this but for now can we remove it.